bin/install-qa-check.d/60openrc

```diff
@@ -33,6 +33,18 @@ openrc_check() {
 			done
 		done
 	fi
+
+	# Make sure deprecated #!/sbin/runscript isn't being used:
+	# Note: $ED contains trailing slash
+	if [[ -d "${ED}"etc/init.d ]] ; then
+		shopt -s nullglob
+		for x in "${ED}"etc/init.d/* ; do
+			if [[ $(head -n1 "${x}") =~ ^#!/sbin/runscript$ ]] ; then
```

Comment:
Maybe it makes sense to use plain == comparison here, since the regex doesn't really gain us anything at this time.